### DISCUSSION TITLE
Compiling TFT_eSPI library outside IDE

### DISCUSSION THREAD

**Discussion by LLMuc** [Bodmer/TFT_eSPI#2947] [Q&A - General]
Dear all,
Thanks for this great library. I have been successfully using it in Arduino IDE and platform.io for STM32.
However, I would like to compile the library and use it in an STM32 Cube IDE project. So far I have been unsuccessful with that.
Is there a Makefile that I can use to compile the library? Can you help me integrate it into an STM32 Cube IDE project?
Thanks, Kind Regards

**Comment by Bodmer** [comment]
Hi, unfortunately I am not familiar with the Cube IDE and thus am unable to assist. Maybe there is a Cube forum that can help.

I will label this as help wanted, maybe someone else has the knowledge required.

**Reply by N0ury** [reply]
@LLMuc This reply is out of the scope of this project.
Why don't you use TouchGFX Designer with Cube IDE?
